src/lib/mdns/minimal/Server.cpp

```diff
@@ -127,7 +127,7 @@ void ServerBase::Shutdown()
     {
         if (mEndpoints[i].udp != nullptr)
         {
-            mEndpoints[i].udp->Close();
+            mEndpoints[i].udp->Free();
```

Comment:
Nice catch.

Reply:
It was Michael - he noticed a error that was hidden for me through all other messages. Apparently after I made mDNS restart on every ip address get, this became more likely to happen.